code/nodes/dataCheck/utils.q

```diff
@@ -110,13 +126,15 @@ dataCheck.i.customPath:{[config]
 
 // @kind function
 // @category dataCheckUtility
-// @fileoverview Construct saved logged file path
-// @param config {dict} Configuration information assigned by the user and related to the current run
-// @return {str} Path constructed to log file based on user defined paths
+// @desc Construct saved logged file path
+// @param config {dictionary} Configuration information assigned by the user 
+//   and related to the current run
+// @return {string} Path constructed to log file based on user defined paths
 dataCheck.i.logging:{[config]
   if[0~config`saveOption;
     if[`~config`loggingDir;
-      -1"\nIf saveOption is 0 and loggingDir is not defined, logging is disabled.\n";
+      -1"\nIf saveOption is 0 and loggingDir is not defined,",
+        " logging is disabled.\n";
     .automl.utils.printing:1b;.automl.utils.logging:0b;:config]];
```

Comment:
Should be indented to match above statement.
`.automl.utils.logging:0b` should be on the next line
`:config]` should be on a separate line
The final `]` should be on a separate line